devicon.json

```diff
@@ -388,6 +388,25 @@
         "color": "#e25a1c",
         "aliases": []
     },
+    {
+        "name": "apl",
+        "altnames": [],
```

Comment:
```suggestion
        "altnames": ["A Programming Language"],
```

Reply:
Thank you!
I've totally missed that info, because I've thought "A Programming Language" was only the book title.
> (...) Then one day Adin Falkoff walked into Ken's office and wrote "A Programming Language" on the board, and underneath it the acronym "APL". (...)

All done!

Reply:
Yeah, I didn't know either before googling `apl (programming language) abbreviation` to which this was the first result: 
> APL, computer programming language based on (and named with the initials of) the book A Programming Language (1962) by Kenneth E. Iverson of IBM.